packages/website/docs/reverseHighlightHit.md

```diff
@@ -2,36 +2,84 @@
 id: reverseHighlightHit
 ---
 
-Returns a virtual node with non-matching parts of an Algolia hit.
+Returns virtual nodes with highlighted non-matching parts of an Algolia hit.
 
-## Example
+ The `reverseHighlightHit` function lets you turn an Algolia hit into a virtual node with highlighted non-matching parts for a given attribute.
+
+## Examples
+
+### With a single string
+
+To determine what attribute to parse, you can pass it as a string.
 
 ```js
 import { reverseHighlightHit } from '@algolia/autocomplete-js';
 
-const hit = {}; // fetch an Algolia hit
-const highlightedValue = reverseHighlightHit({
+// An Algolia hit for query "zelda"
+const hit = {
+  query: 'zelda switch',
+  _highlightResult: {
+    query: {
+      value:
+        'zelda __aa-highlight__switch__/aa-highlight__',
```

Comment:
With the query "zelda", this is the hit that Algolia would return:

```suggestion
        '__aa-highlight__zelda__/aa-highlight__ switch',
```

Reply:
🤦 Yes. Brain malfunction.